lib/services/android-project-service.ts

```diff
@@ -422,16 +403,6 @@ class AndroidProjectService implements IPlatformProjectService {
 		}).future<void>()();
 	}
 
-	private checkJava(): IFuture<void> {
```

Comment:
Yes, it is unused. We used to check for java because we needed to run it manually. Now, we do not this.